tools/wptrunner/wptrunner/wptrunner.py

```diff
@@ -285,6 +285,10 @@ def run_tests(config, test_paths, product, **kwargs):
                                                                   run_info,
                                                                   **kwargs)
 
+                    # TODO(before submitting): Find a better place to set this.
```

Comment:
This TODO is odd.

Reply:
It's highlighting the thing I want opinions on, in this code review, before submitting.

However I think I've just found the place this should go - it looks like we do have a sort-of inheritance system for `get_executor_kwargs`, in the form of **most** products doing `from ..executors import executor_kwargs as base_executor_kwargs` and then calling that function... so as long as that is done consistently, I'll move this to the `executor_kwargs` function in `tools/wptrunner/wptrunner/executors/base.py`.

Reply:
Moved.